source/base/Text.ooc

```diff
@@ -182,6 +182,16 @@ Text: cover {
 		printf("%s\n", this _buffer _backend pointer as Char*)
 		this free(Owner Receiver)
 	}
+	format: func (args: ...) -> This {
+		t := this take()
```

Comment:
This is maybe a bit unnecessary since you only use `t` in one place, better just use `this take()` explicitly perhaps?